.github/dependabot.yml

```diff
@@ -24,6 +24,9 @@ updates:
           - 'react-dom'
           - '@types/react'
           - '@types/react-dom'
+      eslint:
+        patterns:
+          - '*eslint*'
```

Comment:
This worries me as potentially too broad. Can we just make it `eslint*` with the same practical results and less risk of sweeping up something unrelated?

Reply:
I copied the decisions made in the `service` repo and don't think it makes sense to have different rules for the same intention.

See the conversation around this change from there: https://github.com/PhilanthropyDataCommons/service/pull/592#discussion_r1393432857

What do you think?

Reply:
Perfect! I was going to ask if it was too much trouble to do precisely that check. (I do still think `eslint*` is better than `*eslint*`, but we're arguing over trivial risk here. Still, that seems to be what you actually searched for, right?)

Reply:
I don't remember (I think that's a typo and I checked for both) -- but keep in mind there are eslint packages that definitely *do not* start with eslint (e.g. the typescript-namespaced eslint packages) https://github.com/PhilanthropyDataCommons/service/blob/4e0aacb74207eecdf15c6b7869d379ca41ad17c0/package.json#L54

We'll be using those when we pull out of react app, so I do think there is a practical reason to include it.